build-system/tasks/check-types-build-system.js

```diff
@@ -0,0 +1,22 @@
+const { execOrThrow } = require('../common/exec');
```

Comment:
Add copyright and the `use strict;` header. If you [enable auto-fix-on-save for VSCode](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#workflow-for-visual-studio-code), this will automatically happen as you edit files.